tests/unit/test_app.py

```diff
@@ -77,6 +78,49 @@ def name(name):
     return demo
 
 
+def test_mixed_content_types():
+    demo = app.Chalice('demo-app')
+
+    with pytest.raises(ValueError):
+        @demo.route('/index', content_types=['application/octet-stream',
+                                         'text/plain'])
```

Comment:
Yea I manually run it now and then. I've caught quite a few mistakes in the existing tests, but I don't consistently remember to run it.